package.json

```diff
@@ -111,7 +111,7 @@
   "scripts": {
     "tsc": "tsc",
     "tslint": "tslint --project tsconfig.json ./*.ts ./lib/**/*.ts ./test/**/*.ts ./types/**/*.ts",
-    "test": "npm run tsc && npm run tslint && nyc mocha ./dist/test/**/*.js -t 10000 --reporter mocha-junit-reporter --reporter spec",
+    "test": "npm run tsc && npm run tslint && jest --ci --reporters=default --reporters=jest-junit",
```

Comment:
We also need to report to test-results.xml so Azure DevOps recognize it.

Reply:
i think we have the same config as in the portal and there it gets picked up, maybe something else needs to be done if it doesnt show now

Reply:
done